sources/AutoFilerTab.cpp

```diff
@@ -39,8 +46,8 @@ AutoFilerTab::AutoFilerTab()
 	BMessage panelMsg(MSG_FOLDER_CHOSEN);
 	fFilePanel = new BFilePanel(B_OPEN_PANEL, NULL, NULL,
 		B_DIRECTORY_NODE, true, &panelMsg, fRefFilter);
-	fFilePanel->Window()->SetTitle("AutoFiler: Add folders");
-	fFilePanel->SetButtonLabel(B_DEFAULT_BUTTON, "Add");
+	fFilePanel->Window()->SetTitle(B_TRANSLATE("AutoFiler: Add folders"));
+	fFilePanel->SetButtonLabel(B_DEFAULT_BUTTON, B_TRANSLATE("Add"));
```

Comment:
It may be a good idea to use B_TRANSLATE_COMMENT when it comes to button labels.
Button labels should be as short as possible. In German, for example, I can translate "Add" as "Hinzufügen" or "Hinzu". If I know where a string appears in the GUI, I can decide to use "Hinzu" for buttons and "Hinzufügen" for menus, where there's more space.
So, maybe: fFilePanel->SetButtonLabel(B_DEFAULT_BUTTON, B_TRANSLATE_COMMENT("Add", "Used as button in a file panel));